### ISSUE TITLE
Error in jive when I try to monitor a device

### ISSUE THREAD

**Issue by aureocarneiro** [tango-controls/TangoSourceDistribution#25]
In this current release, at MAX IV, when I click on "monitor device", I have the error below. When I directly start atkpanel I have no problem. This class atkpanel/MainPanel is not contained in jive. Is it a release issue or not?

Exception in thread "AWT-EventQueue-0" java.lang.NoClassDefFoundError: atkpanel/MainPanelException in thread "AWT-EventQueue-0" java.lang.NoClassDefFoundError: atkpanel/MainPanel
at jive.JiveUtils.launchAtkPanel(JiveUtils.java:729)
at jive3.TreePanelServer$DeviceServerNode.execAction(TreePanelServer.java:812)
at jive3.TreePanel$11.actionPerformed(TreePanel.java:220)
at javax.swing.AbstractButton.fireActionPerformed(Unknown Source)
at javax.swing.AbstractButton$Handler.actionPerformed(Unknown Source)
at javax.swing.DefaultButtonModel.fireActionPerformed(Unknown Source)
at javax.swing.DefaultButtonModel.setPressed(Unknown Source)
at javax.swing.AbstractButton.doClick(Unknown Source)
at javax.swing.plaf.basic.BasicMenuItemUI.doClick(Unknown Source)
at javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(Unknown Source)
at java.awt.Component.processMouseEvent(Unknown Source)
at javax.swing.JComponent.processMouseEvent(Unknown Source)
at java.awt.Component.processEvent(Unknown Source)
at java.awt.Container.processEvent(Unknown Source)
at java.awt.Component.dispatchEventImpl(Unknown Source)
at java.awt.Container.dispatchEventImpl(Unknown Source)
at java.awt.Component.dispatchEvent(Unknown Source)
at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
at java.awt.Container.dispatchEventImpl(Unknown Source)
at java.awt.Window.dispatchEventImpl(Unknown Source)
at java.awt.Component.dispatchEvent(Unknown Source)
at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
at java.awt.EventQueue.access$500(Unknown Source)
at java.awt.EventQueue$3.run(Unknown Source)
at java.awt.EventQueue$3.run(Unknown Source)
at java.security.AccessController.doPrivileged(Native Method)
at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
at java.awt.EventQueue$4.run(Unknown Source)
at java.awt.EventQueue$4.run(Unknown Source)
at java.security.AccessController.doPrivileged(Native Method)
at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
at java.awt.EventQueue.dispatchEvent(Unknown Source)
at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
at java.awt.EventDispatchThread.run(Unknown Source)Caused by: java.lang.ClassNotFoundException: atkpanel.MainPanel
at java.net.URLClassLoader.findClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
at sun.misc.Launcher$AppClassLoader.loadClass(Unknown Source)
at java.lang.ClassLoader.loadClass(Unknown Source)
... 41 more

**Comment by mliszcz** [comment]
Hi @aureocarneiro 

Following patch solves the issue for me (the fille is installed to `${PREFIX}/bin/jive` so you can patch that one as well):
```diff
diff --git a/assets/lib/java/jive.in b/assets/lib/java/jive.in
index 468aa77..9db2ad3 100644
--- a/assets/lib/java/jive.in
+++ b/assets/lib/java/jive.in
@@ -11,7 +11,7 @@ fi
 JAVALIB=@prefix@/share/java;
 JAVABIN=@prefix@/share/java;

-ATK=$JAVALIB/ATKCore.jar:$JAVALIB/ATKWidget.jar:$JAVABIN/atkpanel.jar
+ATK=$JAVALIB/ATKCore.jar:$JAVALIB/ATKWidget.jar:$JAVABIN/ATKPanel.jar
 LOGVIEWER=$JAVALIB/LogViewer.jar
 ASTOR=$JAVALIB/Astor.jar:$JAVALIB/tools_panel.jar
 LOG4J=$JAVALIB/log4j.jar
```

The issue was introduced in b741d3b559a385c39db5cc6724a5a5dc955c4849 where `atkpanel` was renamed to `ATKPanel`, but some files were not aligned:

```
assets/lib/java/jive.in:ATK=$JAVALIB/ATKCore.jar:$JAVALIB/ATKWidget.jar:$JAVABIN/atkpanel.jar
assets/lib/java/astor.in:ATK_CLASS=$ATK_CLASS:$APP_DIR/atkpanel.jar:$APP_DIR/tool_panels.jar
assets/lib/java/devicetree.in:CLASSPATH=$JAVALIB/JTango.jar:$JAVALIB/ATKWidget.jar:$JAVALIB/ATKCore.jar:$JAVABIN/Jive.jar:$JAVABIN/atkpanel.jar:$JAVABIN/DeviceTree.jar
assets/lib/java/jdraw.in:TANGOATK=$LIB_DIR/ATKCore.jar:$LIB_DIR/ATKWidget.jar:$LIB_DIR/atkpanel.jar
assets/lib/java/synopticappli.in:ATKPANEL=$APP_DIR/atkpanel.jar
```

**Comment by bourtemb** [comment]
Thanks @aureocarneiro for the bug report.
This has been fixed by @mliszcz in #27 and has been merged in prepare-9.3.3 branch which is the development branch for the 9.3.3 release of the Tango Distribution.
So this fix will be available in 9.3.3 release.

**Comment by aureocarneiro** [comment]
@bourtemb and @mliszcz , Thank you so much !
